packages/components/src/tooltip/index.tsx

```diff
@@ -44,8 +46,29 @@ function Tooltip( props: TooltipProps ) {
 		}
 	}
 
+	const DEFAULT_PLACEMENT = 'bottom';
+
+	// Compute tooltip's placement:
+	// - give priority to `placement` prop, if defined
+	// - otherwise, compute it from the legacy `position` prop (if defined)
+	// - finally, fallback to the DEFAULT_PLACEMENT.
+	let computedPlacement;
+	if ( placement !== undefined ) {
+		computedPlacement = placement;
+	} else if ( position !== undefined ) {
+		computedPlacement = positionToPlacement( position );
+	}
+	computedPlacement = computedPlacement || DEFAULT_PLACEMENT;
+
+	if ( position !== undefined ) {
+		deprecated( '`position` prop in wp.components.tooltip', {
```

Comment:
We'll either need to update some tests to expect the `deprecated` call, or to stop using `position` anywhere, otherwise a bunch of tests will fail.

Reply:
Sounds good! I've started to replace our internal use of `position`, but I've run into a few things along the way ([example](https://github.com/WordPress/gutenberg/pull/54263)). In the meantime, I've set this as a draft while I look into them.

Reply:
Sure, just wanted to point it out. Thanks!

Reply:
Good thing you did, as we're going to update the tests in this PR and then replace internal use cases in the next PR. 😄